test/plugins/validation/oas3/security-definitions-ibm.js

```diff
@@ -0,0 +1,235 @@
+const expect = require('expect');
+const {
+  validate
+} = require('../../../../src/plugins/validation/oas3/semantic-validators/security-definitions-ibm');
+
+describe('it should have a type of `apiKey`,`http`,`oauth2`, `openIdConnect`', function() {
+  it('should have a type', function() {
+    const spec = {
+      components: {
+        securitySchemes: {
+          SecuritySchemeModel: {
+            scheme: 'basic'
+          }
+        }
+      }
+    };
+
+    const res = validate({ resolvedSpec: spec });
+    expect(res.errors.length).toEqual(1);
+    expect(res.warnings.length).toEqual(0);
+    expect(res.errors[0].message).toEqual(
+      'security scheme is missing required field `type`'
+    );
```

Comment:
I apologize that I'm just catching this now... will you add assertions for the path returned for all of your tests? For example,
```js
expect(res.errors[0].path).toEqual('whatever.path)
```